src/components/content-collapse/content-collapse.js

```diff
@@ -49,7 +49,7 @@ function ContentCollapse({ label, labelClose, children }) {
         href="#"
         type="button"
         onClick={collapseToggle}
-        className="read-more mt-3 d-inline-flex align-items-center text-decoration-none"
+        className="read-more mt-3 d-inline-flex align-items-center text-decoration-none btn btn-dropdown dropdown-toggle"
```

Comment:
Maybe we can clean also the classes d-inline-flex align-items-center and text-decoration-none now that we have the btn-ones. We have to try (tomorrow?)